worlds/spire/docs/slay-the-spire_en.md

```diff
@@ -9,6 +9,24 @@ For steam-based installation, subscribe to the following mods:
 - Archipelago Multiworld Randomizer Mod from
   the [Slay the Spire Workshop](https://steamcommunity.com/sharedfiles/filedetails/?id=2596397288)
 
+- For GOG or Xbox PC Game Pass installation:
+  1. Download the official Steam Console Client [SteamCMD](https://steamcdn-a.akamaihd.net/client/installer/steamcmd.zip)
+  2. Unpack that .zip file into some folder, right click steamcmd.exe and choose "Run as administrator".
+  3. The client will now update itself. When it's ready type `login anonymous`. Now you are ready to download the actual
+      mods.
+  4. Mod the Spire: Type `workshop_download_item 646570 1605060445` in the SteamCMD window and hit Enter.
```

Comment:
Could we change this list to add the downfall mod as well marking it as optional? I also feel like it's very unnecessary verbose, and could be condensed down to something like:
```
  4. Run the following commands to download the required mod files:
   - Mod the Spire: `workshop_download_item 646570 1605060445`
   - BaseMod: `workshop_download_item 646570 1605833019`
   - ArchipelagoMW: `workshop_download_item 646570 2596397288`
   - (optional) Downfall: -insert downfall command here-
```

Reply:
Yeah I like that better.

Reply:
Unfortunately, Downfall isn't supported on GOG/Game Pass at all. It's not distributed as a Steam Workshop package, but instead as a Steam "Community-Made Mod" which requires you to own the base game on Steam to download it.

> Steam is the only way we can ensure that all users of Downfall already own Slay the Spire. Our version is effectively everything that the original game has, with added content, and since we are a free download, we'd effectively be giving away the base game. And we can't have that

https://steamcommunity.com/app/1865780/discussions/0/3186866558190995357/

I will note in the instructions that Downfall is not supported for GOG / Game Pass

Reply:
https://steamcommunity.com/sharedfiles/filedetails/?id=1610056683&searchtext=downfall

it is actually on the workshop, and this is the only version that my mod supports, the steam store page one that you linked does not work with any other mods at all.

Reply:
Oh, nice! That should work then